torch/_inductor/lowering.py

```diff
@@ -1911,7 +1911,11 @@ def unsupported_input_tensor(t: torch.Tensor, parent=None, node=None):
             and parent.target
             in (
```

Comment:
The parent vs node thing does not really make any sense to me, and that is where the bug is currently. It was added in https://github.com/pytorch/pytorch/commit/6977ba6e3c70def3532a44e9ebcb1c6b3d257a8d but not working correctly for our case. We should only be look at the node.